Tools/AP_Periph/Parameters.cpp

```diff
@@ -43,26 +43,57 @@ extern const AP_HAL::HAL &hal;
 const AP_Param::Info AP_Periph_FW::var_info[] = {
     // @Param: FORMAT_VERSION
     // @DisplayName: Eeprom format version number
-    // @Description: This value is incremented when changes are made to the eeprom format
+    // @Description: This value represents the current version number of the eeprom struture format. Set to zero to erase eeprom and perform a factory reset.
```

Comment:
spelling structure

Reply:
fixed. I changed the text back to match the other vehicles